src/components/application_manager/src/resumption/resumption_data_json.cc

```diff
@@ -283,6 +284,60 @@ uint32_t ResumptionDataJson::GetIgnOffTime() const {
   return resumption[strings::last_ign_off_time].asUInt();
 }
 
+uint32_t ResumptionDataJson::GetGlobalIgnOnCounter() const {
+  using namespace app_mngr;
+  LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock autolock(resumption_lock_);
+  Json::Value& resumption = GetResumptionData();
+  if (!resumption.isMember(strings::global_ign_on_counter)) {
+    resumption[strings::global_ign_on_counter] = 1;
```

Comment:
@AByzhynar it is not expected that getter will change data.

Reply:
@LuxoftAKutsan Absolutely agree

Reply:
@LuxoftAKutsan Done in 93ff02d145e8fd2ab3f2ab263d75b6d229fe34d6